build/template-run-integration-tests.yaml

```diff
@@ -14,6 +14,9 @@ steps:
     rerunMaxAttempts: '3'
     runInParallel: false
     codeCoverageEnabled: true
+    failOnMinTestsNotRun: true
+    minimumExpectedTests: '1'
+    pathtoCustomTestAdapters: 'C:\temp' # Workaround for test failure, as NUnit Test Adapter that gets detected seems to screw something up
```

Comment:
@bgavrilMS @trwalke Why do we need NUnit Adapter anyway (since we use MS Test)? Only for mobile tests? Wondering what effect this would have on tests that need the adapter?

Reply:
@pmaytak I think it's for mobile tests, yes. But the default path for Test Adapter discovery as that i searches ALL code for it (and mobile test projects are also deployed) - We could just probably separate the projects and not deploy the mobile tests here as another solution

Reply:
Well all tests including Android seem to pass, so this is good. 👍